pkg/e2e/e2e.go

```diff
@@ -30,6 +30,10 @@ import (
 	"github.com/jetstack/kube-oidc-proxy/pkg/util"
 )
 
+const (
+	readinessProbeURL = "http://127.0.0.1:8080/ready"
```

Comment:
Ideally we should use `util.FeePort()` here but need [this PR](https://github.com/jetstack/kube-oidc-proxy/pull/88). Will fix that up in a later PR.